renderer/src/features/registry-servers/components/grid-cards-registry.tsx

```diff
@@ -68,10 +68,17 @@ export function GridCardsRegistry({
             key={item.name}
             item={item}
             onClick={() => {
-              navigate({
-                to: '/registry/$name',
-                params: { name: item.name! },
-              })
+              if (item.type === 'group') {
+                navigate({
+                  to: '/registry-group/$name',
+                  params: { name: item.name! },
+                })
+              } else {
+                navigate({
+                  to: '/registry/$name',
+                  params: { name: item.name! },
+                })
+              }
```

Comment:
the only change is on `to`

```suggestion
            navigate({
                  to: item.type === 'group' ? '/registry/$name' : '/registry-group/$name',
                  params: { name: item.name! },
                })
```